code/src/pages/products/ProductsPage.tsx

```diff
@@ -29,10 +30,14 @@ const ProductsPage = () => {
     isError
   } = useGetProductsQuery({
     page: page - 1,
-    size: 10,
+    size: Infinity,
```

Comment:
Same question

Reply:
This is just a temporary solution. It is made so, i can pull list of all products from db, because currently db requires size of the elements that we get from DB. When we implement endless scrolling, then we will go back to using a realistic value for that DB request.